content/services/index.njk

```diff
@@ -13,9 +13,27 @@ summary: |
 ---
 
 {% import 'post.macros.njk' as post %}
+{% import 'embed.macros.njk' as embed %}
 
-{{ post.grid(
-  pages=services,
-  level=2,
-  collections=collections
-) }}
+{% for item in services %}
+  {% set page = collections.all | getPage(item) %}
+    <article {{ post.aria_label(page) }} data-work="summary">
+      <div class="offset-grid">
+        <div class="offset-item offset-media">
+            {% set image = page.data.image %}
+              {{ embed.img(
+                src=image.src,
+                alt=image.alt
+              ) }}
+        </div>
+        <div class="offset-item project-details">
+        <header class="post-header">
+          {% call post.banner(page, level=2,called=true) -%}
+           {{  page.data.title | mdInline | safe }}
+          {%- endcall %}
+        </header>
+          <p>{{ page.data.summary | md | elide | safe }}</p>
```

Comment:
Could we get away with dropping `| elide` here? Are the summaries so long that we need to trim crop them?

Reply:
Yeah, "web design & development" and "research & concepting" are both almost twice as long as the other two services, so without the `elide` filter they look really tall. 

![bild](https://user-images.githubusercontent.com/41588129/105986751-9f14bb00-609d-11eb-9cb0-199090cfbf34.png)

![bild](https://user-images.githubusercontent.com/41588129/105986825-b5227b80-609d-11eb-86ed-94be57cb8e28.png)

Reply:
makes sense - yeah, good call.